src/library/Uno.Material/Styles/Application/v1/ColorPalette.xaml

```diff
@@ -50,6 +50,8 @@
 			<SolidColorBrush x:Key="MaterialBottomNavUncheckedForegroundBrush"
 							 Color="{ThemeResource MaterialOnPrimaryColor}"
 							 Opacity="0.60" />
+			
+			<Color x:Key="MaterialShadowColor">#33000000</Color>
```

Comment:
We'll either need to add this as a "ShadowColor" in the [SharedColorPalette](https://github.com/unoplatform/Uno.Themes/blob/master/src/library/Uno.Material/Styles/Application/v2/SharedColorPalette.xaml) or create lightweight styling keys that can be used in the material styles for each of the v2 control styles that need it

Reply:
Put it as "ShadowColor" in SharedColorPalette, is that sufficient or something else needs to be done please 😅 ?